### DISCUSSION TITLE
Using emojis with driver.type

### DISCUSSION THREAD

**Discussion by benfrankie** [seleniumbase/SeleniumBase#2814] [Q&A] [answered]
Hi, I'm encountering an issue when using driver.type() with text containing emojis. The error message I receive is: "ChromeDriver only supports characters in the BMP," which I believe is due to the presence of emojis in the text.

Here's a snippet of the code that's causing the issue:

generated_reply = "Hello! 😊 How can I assist you today?"
driver.type("#new-message-textarea", generated_reply)

Is there a way to handle emojis and other non-BMP characters with driver.type() or any workaround you would recommend to avoid triggering detection while entering text in a textarea?

Thank you for your help!

**Comment by mdmintz** [comment, marked as answer]
Convert to unicode first. There's more info on Stack Overflow: https://stackoverflow.com/a/61043442/7058266
